packages/flutter_test/test_fixes/flutter_test/binding/test_widgets_flutter_binding.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:clock/src/clock.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  TestWidgetsFlutterBinding binding = AutomatedTestWidgetsFlutterBinding.ensureInitialized();
+  binding.runTest(
+    () async {
+      // This will be unchanged as there is no equivalent API.
+      binding.addTime(Duration(seconds: 30));
```

Comment:
Come to think of it: These calls should really just be removed with no replacement. They are currently no-ops anyways. Can dart fix do that?

Reply:
Dart fix doesn't have a `removeCall` ability unfortunately. It'd be pretty hard to do in any case more complex than this, so it kinda makes sense that it's not supported, but still sad.

Reply:
Might be worthwhile to file a feature request for this over at https://github.com/dart-lang/sdk/issues/new/choose. Even if it just supports removal in the simplest of all cases (stand-alone statements like this one) it could be pretty useful.

Reply:
@goderbauer

For the record, I didn't create an issue specifically for adding remove capabilities to dart fix, but I did file [dart #51987](https://github.com/dart-lang/sdk/issues/51987) to cover a broader use case of "things we can't automate fixes for at the moment".